src/Microsoft.VisualStudio.Editors/PropPages/DebugPropPage.vb

```diff
@@ -135,13 +132,6 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                 Me.rbStartProgram.Checked = (action = VSLangProj.prjStartAction.prjStartActionProgram)
                 Me.rbStartURL.Checked = (action = VSLangProj.prjStartAction.prjStartActionURL)
                 Me.StartURL.Enabled = (action = VSLangProj.prjStartAction.prjStartActionURL)
-
-                If action = VSLangProj.prjStartAction.prjStartActionProject Then
-                    EnableControl(UseVSHostingProcess, True)
```

Comment:
I won't dare ask why EnableControl is used to enable controls but you don't need it to disable controls.